metadata/commons-logging/commons-logging/index.json

```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "latest": true,
```

Comment:
Should we use the new `default-for` flag too?

Reply:
If we have metadata for only one version, how we would set `default-for`regexp? To which versions we should limit the  metadata? The existing latest metadata might be appropriate for future versions too, so I think we should not limit versions for latest metadata until metadata for a new version is created. @vjovanov @fniephaus - your thoughts?

Reply:
I will merge these changes to the master branch since I need it for other tests, but I guess we can continue discussion here for future PRs.